example/http/client/async-ssl/http_client_async_ssl.cpp

```diff
@@ -229,7 +244,7 @@ int main(int argc, char** argv)
     ctx.set_verify_mode(ssl::verify_peer);
 
     // Launch the asynchronous operation
-    std::make_shared<session>(ioc, ctx)->run(host, port, target, version);
+    std::make_shared<session>(exec, ctx)->run(host, port, target, version);
```

Comment:
Wouldn't this make more sense as
```
std::make_shared<session>(
    net::make_strand(ioc.get_executor()),
    ctx
        )->run(host, port, target, version);
```

Reply:
I think it would to you. To me it looks like a horrid mess :-)

Reply:
Which part is the mess, the form or the substance?

Reply:
for me, the form. But then as you know I like to line up my soldiers.